apps/clinical/src/components/forms/immunizationHistory/styles/ImmunizationHistoryForm.module.scss

```
@@ -52,6 +52,15 @@
   margin-left: 0;
 }
 
+.batchNumberComboBox :global(.cds--list-box__menu-item__option) {
+  overflow: visible;
+  text-overflow: unset;
+  white-space: normal;
+  word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)